docs/api/endorsement-provisioning/endorsement-provisioning.yaml

```
@@ -0,0 +1,71 @@
+openapi: '3.0.0'
+
+info:
+  title: attestation endorsement provisioning
+  description: For further details, see datatracker.ietf.org/doc/draft-birkholz-corim
```

not sure this applies:

```suggestion
  description: For further details, see datatracker.ietf.org/doc/draft-birkholz-corim
```

---

Add a description and fix the URL